src/hooks/UseMediaQuery.test.tsx

```diff
@@ -0,0 +1,26 @@
+import { expect, it, describe } from "vitest";
+import { renderHook } from "@testing-library/react-hooks";
+import useMediaQuery from "./use-media-query";
+
+describe("useMediaQuery", () => {
+  beforeAll(() => {
```

Comment:
I think here we should not need this

Reply:
@Mubashar1009  The `beforeAll` block is currently used to mock the matchMedia function, which allows us to simulate different screen sizes in our tests."